workbench-core/datasets/src/wbcDataSetsAuthorizationPlugin.ts

```diff
@@ -17,7 +26,52 @@ export class WbcDataSetsAuthorizationPlugin implements DataSetsAuthorizationPlug
   }
 
   public async addAccessPermission(params: AddRemoveAccessPermissionRequest): Promise<PermissionsResponse> {
-    throw new Error('Method not implemented.');
+    if (params.permission.accessLevel !== 'read-only' && params.permission.accessLevel !== 'read-write') {
```

Comment:
Maybe extract `read-only` and `read-write` into constants since they're used frequently?

Reply:
you could also future proof this a little bit if you create an array of permission values and then do something like `if (!permissionsValues.contains(accessLevel))`

Reply:
interesting. If I use generic string constants, the compiler will reject as the type is a Union of two literals. I suppose I could define the literals.